scripts/work_order_queue.py

```
@@ -0,0 +1,291 @@
+        deadline = time.monotonic() + timeout_seconds
+        while time.monotonic() < deadline and not trade.isDone():
+            ib.waitOnUpdate(timeout=1.0)
+            sync_trade_progress(session, order, trade, event_type="order_progress")
+            session.flush()
+
+        sync_trade_progress(session, order, trade, event_type="order_final")
+        if trade.advancedError:
+            order.last_error = trade.advancedError
+            order.updated_at = now_utc()
+            append_order_event(session, order, "ib_advanced_error", trade.advancedError)
+    except Exception as exc:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)